scripts/eval/yamls/eval_gauntlet.yaml

```diff
@@ -2,6 +2,27 @@ eval_gauntlet:
   weighting: EQUAL
   subtract_random_baseline: true
   rescale_accuracy: true
+  averages:
```

Comment:
While this is extremely configurable, the challenge is still that "core_average" as it shows up in w&b could still mean many things if someone accidentally excluded (or added) a category in the yaml construction.

I wonder if we should be hard-coding some of these?

Reply:
I think this is something we should discuss for future re-factoring @dakinggg